hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -650,6 +654,18 @@ public void syncHive(HiveConf conf) {
     syncHive();
   }
 
+  private void syncGlue() {
```

Comment:
We need to keep `DeltaSync` free of any one catalog sync. Can't this work via reflection?